components/loading.tsx

```diff
@@ -11,6 +11,8 @@ const variantTexts = {
 	clock: ['🕐 ', '🕑 ', '🕒 ', '🕓 ', '🕔 ', '🕕 ', '🕖 ', '🕗 ', '🕘 ', '🕙 ', '🕚 '],
 	pipe: ['\\', '|', '/', '-'],
 	point: ['⠙', '⠘', '⠰', '⠴', '⠤', '⠦', '⠆', '⠃', '⠋', '⠉'],
+	wave: [':...', '.:..', '..:.', '...:'],
+	wave: [':...', '.:..', '..:.', '...:','..:.', '.:..',':...'],
```

Comment:
```suggestion
	longwave: [':...', '.:..', '..:.', '...:','..:.', '.:..',':...'],
```
I think this is better